zio-cli/shared/src/main/scala/zio/cli/ParserOptions.scala

```diff
@@ -23,5 +25,5 @@ object ParserOptions {
   /**
    * The default options are case sensitive parsing
    */
-  val default: ParserOptions = ParserOptions(false)
+  val default: ParserOptions = ParserOptions(false, 2)
```

Comment:
Did you experiment with options here?  Also, it occurs to me, we should not attempt auto-correct if the option name is too short, e.g. `-c` cannot really benefit from auto-correct.

Reply:
I played around with this and saw that a value of 2 or 3 is good since from experience with using cli apps most option names are short.

Reply:
Sounds good.